### DISCUSSION TITLE
Will the "unzip" feature be (re)implemented?

### DISCUSSION THREAD

**Discussion by denehoffman** [jax-ml/oryx#59] [Q&A]
I just came across this project a few days ago and I'm really interested in using the `harvest` functions in a few projects of mine. I was also interested in the `unzip` function which is mentioned in some but not all of the docs, but doesn't seem to actually be a part of the current project. Was this feature removed, or has it not actually been implemented, or was it moved into another project?

**Comment by sharadmv** [comment]
Unzip was part of Oryx but was removed because most of its functionality was subsumed by harvest (you can implement unzip using harvest) and its implementation relied on a lot of JAX internals that changed.
